src/backend/partaj/core/indexers/common.py

```diff
@@ -94,3 +94,26 @@ def partaj_bulk(actions):
         client=ES_CLIENT,
         stats_only=True,
     )
+
+
+DEFAULT_DELIMITERS = [" ", "/", "|"]
+
+
+def slice_string_for_completion(string, delimiters=None):
```

Comment:
Maybe put DEFAULT_DELIMITERS as default for delimiters

Reply:
`pylint` does not like lists as default arguments because they're actually long-lived values that will be reused by the function over multiple calls, but they look like a brand new list created a function call time.

The way I did it does not prevent the issue but makes it obvious mutating the list would be changing up a constant. And `pylint` does not complain 😅